### ISSUE TITLE
Keeping AwesomeLists things together

### ISSUE THREAD

**Issue by rwese** [guardrailsio/awesome-golang-security#1]
Imo it would be a good idea to add this list to the already existing awesomelist (https://github.com/sindresorhus/awesome you also listed) to not fragment the information.

**Comment by kytwb** [comment]
Good call! What about creating a subsection to the [Security](https://github.com/sindresorhus/awesome#security) section via a pull request that lists the security-related awesome list per language? So far I can see one for Golang, [Node.js](https://github.com/lirantal/awesome-nodejs-security), [PHP](https://github.com/guardrailsio/awesome-php-security) and [Ruby](https://github.com/pxlpnk/awesome-ruby-security).

**Comment by streichsbaer** [comment]
Thanks for sharing your thoughts @rwese. 

I agree that we should reference the lists back to the original one. Once the [30 day period](https://github.com/sindresorhus/awesome/blob/94c5231b49fbb467a648be9a12bc3ccb6671da15/create-list.md) required for a list to mature is passed.

**Comment by kytwb** [comment]
@streichsbaer I think we can start taking a look at the [pull request template](https://github.com/sindresorhus/awesome/blob/94c5231b49fbb467a648be9a12bc3ccb6671da15/pull_request_template.md).

**Comment by kytwb** [comment]
We can take it from there: https://github.com/sindresorhus/awesome/compare/master...kytwb:master.
